common-util/DisplayName/index.tsx

```diff
@@ -3,17 +3,10 @@ import PropTypes from 'prop-types';
 
 import { truncateAddress } from 'common-util/functions';
 
-const DisplayName = ({ actorAddress, username }: { actorAddress: string; username: string }) => (
+type DisplayNameProps = {
+  actorAddress: string;
+  username: string;
```

Comment:
looking at the logic, it’s possible that username could be undefined, right?
```suggestion
  username?: string;
```